airflow/ui/src/components/MarkAs/MarkAsAccordion.tsx

```diff
@@ -16,57 +16,95 @@
-          onChange={(event: ChangeEvent<HTMLInputElement>) => setNote(event.target.value)}
-          value={note ?? ""}
-        >
-          <Editable.Preview
-            _hover={{ backgroundColor: "transparent" }}
-            alignItems="flex-start"
-            as={VStack}
-            gap="0"
-            height="200px"
-            overflowY="auto"
-            width="100%"
+const MarkAsAccordion = ({ affectedTasks, note, setNote }: Props) => {
+  const showTaskSection = affectedTasks !== undefined;
+
+  return (
+    <Accordion.Root
+      collapsible
+      defaultValue={showTaskSection ? ["tasks"] : ["note"]}
+      multiple={false}
+      variant="enclosed"
+    >
+      {showTaskSection ? (
+        <Accordion.Item key="tasks" value="tasks">
+          <Accordion.ItemTrigger>
+            <Text fontWeight="bold">Affected Tasks: {affectedTasks.total_entries}</Text>
+          </Accordion.ItemTrigger>
+          <Accordion.ItemContent>
+            <Box maxH="400px" overflowY="scroll">
+              <DataTable
+                columns={columns}
+                data={affectedTasks.task_instances}
+                displayMode="table"
+                initialState={{
+                  pagination: {
+                    pageIndex: 0,
+                    pageSize: affectedTasks.total_entries,
+                  },
+                  sorting: [],
+                }}
+                modelName="Task Instance"
+                total={affectedTasks.total_entries}
+              />
+            </Box>
+          </Accordion.ItemContent>
+        </Accordion.Item>
+      ) : undefined}
+      <Accordion.Item key="note" value="note">
+        <Accordion.ItemTrigger>
+          <Text fontWeight="bold">Note</Text>
+        </Accordion.ItemTrigger>
+        <Accordion.ItemContent>
+          <Editable.Root
+            onChange={(event: ChangeEvent<HTMLInputElement>) => setNote(event.target.value)}
+            value={note ?? ""}
           >
-            {Boolean(note) ? (
-              <ReactMarkdown>{note}</ReactMarkdown>
-            ) : (
-              <Text color="gray" opacity={0.6}>
-                Add a note...
-              </Text>
-            )}
-          </Editable.Preview>
-          <Editable.Textarea
-            data-testid="notes-input"
-            height="200px"
-            overflowY="auto"
-            placeholder="Add a note..."
-            resize="none"
-          />
-        </Editable.Root>
-      </Accordion.ItemContent>
-    </Accordion.Item>
-  </Accordion.Root>
-);
+            <Editable.Preview
+              _hover={{ backgroundColor: "transparent" }}
+              alignItems="flex-start"
+              as={VStack}
+              gap="0"
+              height="200px"
+              overflowY="auto"
+              width="100%"
+            >
+              {Boolean(note) ? (
+                <ReactMarkdown>{note}</ReactMarkdown>
+              ) : (
+                <Text color="gray" opacity={0.6}>
```

Comment:
We probably want to use something like `text.subtle`

<img width="427" alt="Screenshot 2025-01-27 at 12 29 35 PM" src="https://github.com/user-attachments/assets/a2a61dda-d417-4f72-b238-5a28fbea57e0" />

Reply:
Yes, the reason I did that in the first place is because I wanted the 'placeholder' of the markdown rendering to be exactly the same as the placeholder of the `Editable.Textarea` so there's no change when we focus on the item. But  indeed it won't play nice with dark mode.

I switched to `fg.suble`